sparsnas_decode.cpp

```diff
@@ -440,6 +447,12 @@ int main(int argc, char **argv)
     else
       sprintf(MQTT_TOPIC, "sparsnas/%d", SENSOR_ID);
 
+    memset(MQTT_CRC_TOPIC, '\0', sizeof(MQTT_CRC_TOPIC));
+    if (const char *env_p = std::getenv("MQTT_CRC_TOPIC"))
```

Comment:
This is not needed, I think  it is better to let `MQTT_CRC_TOPIC` be:

`sprintf(MQTT_CRC_TOPIC, "%s/crc", MQTT_TOPIC, SENSOR_ID);`

Reply:
This is true.